src/hooks/useAccountAccentColor.ts

```diff
@@ -0,0 +1,26 @@
+import { maybeSignUri } from '@/handlers/imgix';
+import { useTheme } from '@/theme';
+import {
+  useAccountProfile,
+  usePersistentDominantColorFromImage,
+} from '@/hooks';
+
+export function useAccountAccentColor() {
```

Comment:
if the values are cached it should be very quick, what you're bringing up is a really good point but imo is a bigger task. we should be saving accountColor when a user changes their emoji, image, or ens profile, currently that is very much not the case which is why theres so much funky logic here instead of just reaching for accountColor